### DISCUSSION TITLE
Is this layout possible

### DISCUSSION THREAD

**Discussion by ctyar** [spectreconsole/spectre.console#572] [Q&A] [answered]
I recently started using your project. I was wondering if it is possible to create a layout like this:

![image](https://user-images.githubusercontent.com/1432648/96292493-3164e780-0ff6-11eb-8e83-390190fa3a6d.png)

With a Column and two Panels, I can divide the console into two equal boxes.
I tried using two columns and nested Columns but it didn't work

Thanks for your great project.

**Comment by patriksvensson** [comment, marked as answer]
@ctyar Thank you for your kind words.

I guess you could accomplish something like this by using a combination of `Grid`, `Rows` and `Columns`, but I would recommend against it.

From your description, it sounds like you're looking for a TUI framework, and `Spectre.Console` is not a TUI framework, but a library to draw (static) things to the console. Height is not taken into account for example. `Spectre.Console` is perfect for scenarios where you want to draw things like a non-interactive table to display some information to the user as part of a CLI application.

I would recommend that you look into something like `Terminal.Gui` (or GUI.cs as it's also called) for what I think you're trying to accomplish: https://github.com/migueldeicaza/gui.cs
